lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -242,6 +242,10 @@ def find_failed_line
           line_regex = RSpec.configuration.in_project_source_dir_regex
           loaded_spec_files = RSpec.configuration.loaded_spec_files
 
+          exception_backtrace.reject! do |line|
+            line =~ /\A<internal:/
```

Comment:
This is a rubocop failure:
```
lib/rspec/core/formatters/exception_presenter.rb:246:13: C: Performance/StartWith: Use String#start_with? instead of a regex match anchored to the beginning of the string.
            line =~ /\A<internal:/
```